plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -152,9 +154,16 @@ public OpenShiftConnector(DockerConnectorConfiguration connectorConfiguration,
         this.openShiftLivenessProbeDelay = openShiftLivenessProbeDelay;
         this.openShiftLivenessProbeTimeout = openShiftLivenessProbeTimeout;
 
-        Config config = new ConfigBuilder().withMasterUrl(openShiftApiEndpoint)
-                .withUsername(openShiftUserName)
-                .withPassword(openShiftUserPassword).build();
+        Config config;
+        if (StringUtils.isNotBlank(openShiftToken)) {
```

Comment:
You are right, I didn't want to import another package in this class so I used the available one. But maybe we should use guava when possible instead.